WiserTaskScheduler/WiserTaskScheduler/Modules/Body/Services/BodyService.cs

```diff
@@ -38,13 +40,23 @@ public string GenerateBody(BodyModel bodyModel, List<int> rows, JObject resultSe
             foreach (var bodyPart in bodyModel.BodyParts)
             {
                 var body = bodyPart.Text;
+                
+                // If an item-id is set, get the template from the database.
+                if (bodyPart.ItemId > 0)
+                {
+                    // Create a scope to get the string wiserItems service. It's needed to get the template
+                    var wiserItemsService = scope.ServiceProvider.GetRequiredService<IWiserItemsService>();
+
+                    var template = wiserItemsService.GetItemDetailsAsync(bodyPart.ItemId, languageCode:bodyPart.LanguageCode, detailKey:bodyPart.PropertyName, skipPermissionsCheck: true);
```

Comment:
Please add a single space after the colons (`:`). So `languageCode: bodyPart.LanguageCode` instead of `languageCode:bodyPart.LanguageCode`. The same goes for `detailKey`.

Reply:
Added spaces